### PR TITLE
skipper: setup lua environment for operators

### PR DESCRIPTION
**Pull request by szuecs**
skipper: setup lua environment such that operators can create lua files via configmap to be used in routes

### CODE REVIEW

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -109,7 +109,8 @@ skipper_validate_query_log: "false"
 
 skipper_default_filters: 'disableAccessLog(2,3,404,429) -> fifo(2000,20,"1s")'
 skipper_default_filters_append: 'stateBagToTag("auth-user", "client.uid")'
-skipper_disabled_filters: "lua,static,bearerinjector"
+skipper_disabled_filters: "static,bearerinjector"
+skipper_lua_sources: "file"
```
Comment (AlexanderYastrebov) [inline cluster/config-defaults.yaml:113]: Maybe we can reuse existing `skipper_lua_scripts_enabled` instead?

Comment (szuecs) [reply cluster/config-defaults.yaml:113]: 1) why?
2) I don't really understand your proposal

Comment (AlexanderYastrebov) [reply cluster/config-defaults.yaml:113]: We already have `skipper_lua_scripts_enabled` flag that we can check instead of  adding another `skipper_lua_sources`.
Or do you want to have a way to allow lua inline script selectively as well?

**Comment by AlexanderYastrebov** [comment]
:+1:

File: cluster/config-defaults.yaml (hunk 1 above)
Comment (szuecs) [reply cluster/config-defaults.yaml:113]: I need only in 3 clusters lua file and whatever we need in some other edge case.
It feels wrong to me to use `skipper_lua_scripts_enabled` as config.

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
load test has ok enough results
